src/Command/RestoreTableFromImports.php

```diff
@@ -34,18 +40,27 @@ class RestoreTableFromImports extends Command
      */
     private $input;
 
+    /**
```

Comment:
Nebylo by lepší mít "tenké" docblocky?
Settings → Editor → File and Code Templates:
`PHP Field Doc Comment:` → `/** ${TYPE_TAG} ${TYPE_HINT} */`

Reply:
Typicky tam nikdy není nic víc než typ, tak to zabere zbytečně 3x tolik řádků a musí se scrollovat, protože je to vždycky nahoře v třídě

Reply:
Jako mohli bysme, koukám že Guzzle to tak taky používá. Ale asi by bylo fajn kdyby se to tím pádem pak používalo všude.

Reply:
a to rika clovek, kterej pise jednparametrickej constructor na 3 radky :D

Reply:
Konstruktor natahuje třídu lineárně, trojdocblocky exponenciálně 😛

Reply:
Ne, furt lineárně :-)

Reply:
Nj :/ Pravda `y = 3x` :)